src/Views/EncryptView.vala

```diff
@@ -32,9 +32,13 @@ public class EncryptView : AbstractInstallerView {
     }
 
     construct {
-        var image = new Gtk.Image.from_icon_name ("drive-harddisk", Gtk.IconSize.DIALOG);
+        var image = new Gtk.Image.from_icon_name ("drive-harddisk", Gtk.IconSize.INVALID) {
+            pixel_size = 128,
+            valign = Gtk.Align.END
```

Comment:
afaict this valign isn't doing anything here?

Reply:
Hmmm not that I can visibly tell